src/main/java/org/gridsuite/modification/server/impacts/SimpleElementImpact.java

```diff
@@ -6,38 +6,30 @@
  */
 package org.gridsuite.modification.server.impacts;
 
-import com.powsybl.iidm.network.IdentifiableType;
 import lombok.*;
+import lombok.experimental.SuperBuilder;
 
 import java.util.Set;
 
 /**
  * This class describes an element type network impact
- * This type of network impact only describes an individual impacted item and the list of associated subtractions
+ * This type of network impact only describes an individual impacted item and the list of associated substations
  *
  * @author Slimane Amar <slimane.amar at rte-france.com>
+ * @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
  */
 @AllArgsConstructor(access = AccessLevel.PRIVATE)
 @Setter
 @Getter
-@Builder
+@SuperBuilder
 @NoArgsConstructor
-@EqualsAndHashCode(callSuper = false)
+@EqualsAndHashCode(callSuper = true)
 @ToString
```

Comment:
Like before, Lombok annotations can be simplified

Reply:
Is it ok for you now ?